openerp/addons/base/res/ir_property.py

```diff
@@ -47,6 +61,14 @@ class Property(models.Model):
                             required=True,
                             default='many2one',
                             index=True)
+    value = fields.Char(compute='_compute_value')
+
+    @api.constrains('value_reference')
+    def _check_value(self):
+        for prop in self.filtered('value_reference'):
+            pattern = re.compile('([a-zA-Z.]+),(\d+)')
```

Comment:
Well, model name can have _(underscrore) or digits, check the object_name regex we support [here](https://github.com/odoo/odoo/blob/master/openerp/models.py#L62)